Rename to tinted-putty and add Base24 template and themes

JamyGolden opened the pull request:
## Description

We're moving towards supporting multiple scheme systems with our repos [tinted-sublime-text](https://github.com/tinted-theming/tinted-sublime-text), [tinted-vscode](https://github.com/tinted-theming/tinted-vscode)  we're going to move towards this with our other template repositories too (ie [base16-shell](https://github.com/tinted-theming/base16-shell/pull/53) [base16-tmux](https://github.com/tinted-theming/base16-tmux/pull/22) and base16-vim soon) since it's easier to maintain and easier for users to have access to the different themes without having to add multiple scheme system repos per template.

## PR Changes

- Add base24 template and theme support to repo
- Update readme

## Backwards compatibility

Renaming to `tinted-putty` we need to make sure backwards compatibility is there.

The plan is to rename the repo to https://github.com/tinted-theming/tinted-putty and have github auto redirect from https://github.com/tinted-theming/base16-putty to the new repo.

Any user who has `base16-putty` cloned locally should be able to do a `git pull` and carry on without knowing anything has changed aside from getting more themes in `./putty/base24-*`.

JamyGolden commented:
@FredHappyface I'm tagging you here because you have https://github.com/base24/base24-putty. I didn't use that theme because I'm not very familiar with putty and your theme seems to be doing a lot more than the base16 template in this repo. Should we update the base24 template here to match your template?

FredHappyface commented:
I no longer use putty so I must admit I've completely forgotton. I imagine it will have been as I was working a lot on debian at the time so a `.reg` file wouldn't have worked. So I believe the default settings file corresponds to the `~/.putty/` dotfile(s). I would suggest adding both? But maybe we can stick with the `.reg` file for now and evaluate if there's a need for .*nix support

JamyGolden commented:
Ahh makes sense :+1: yeah would be nice to have nix support. I'll add your templates after this PR is merged since there's no reason not to support more systems :shrug:

FredHappyface commented:
Cool hopefully those work as expected. Always happy to take another look later. Though might be worth opening a separate pr for any refinements (if needed)

JamyGolden commented:
@abravalheri are you happy with the changes?

abravalheri commented on templates/base24.mustache:
```diff
@@ -0,0 +1,111 @@
+
+; ANSI Black Bright
+; 1;30m
+; base03 #{{base02-hex}}
```
These comments probably have to be reworked to actually express what is going on...
They seem to be a bit misleading the way they are right now (there is a mismatch between the number in the comment and the variable name)...

abravalheri reviewed (commented):
Hi @JamyGolden, sorry for the delay, I was away for 2 weeks and could only check github on my phone.

Sadly, with 273 files changes I could not scroll down to the template files 😅 on the app.

I think the best here would be to split the PR into 2 (or at list split the same PR into 2+ commits). One with the template changes and another one that adds the rendered files. This would make it easier to navigate the repository history in the future.

Any reason for keeping the `templates/default.mustache` file around? Instead of adding an extra file, should `git mv` be used to create the `templates/base16.mustache` file (also good for keeping history)?

JamyGolden commented:
Thanks for your comments @abravalheri. I've split the commit into 2 and also made sure the comment basexx numbers match the associated value. I've removed the ansi escape code values from the comments in the base24 template for now since it's not that important and one would have to go over each escape sequence to make sure it matches up.
